docs/sender.md

```diff
@@ -209,6 +210,20 @@ compression_level is an integer from 0 to 9 or -1 controlling the level of compr
  (1, each time, if not zipped, 0..X if zipped)
  
  
+### CA_MD_TOO_WEAK
+Or CA signature digest algorithm too weak its a error with news versions of openssl>=1.1.0
+If you have problem with your certificates of Devo and devo-sdk you with this error you can add flag `sec_level=0` 
```

Comment:
Idem as function doc. If we can add some info regarding valid values will be helpful

Reply:
Added more info with all levels.